packages/bot/src/util/handleThreadManagement.ts

```diff
@@ -0,0 +1,99 @@
+import { PrismaClient } from '@prisma/client';
+import {
+	type ChatInputCommandInteraction,
+	Colors,
+	EmbedBuilder,
+	type TextChannel,
+	time,
+	TimestampStyles,
+	type UserContextMenuCommandInteraction,
+} from 'discord.js';
+import i18next from 'i18next';
+import { container } from 'tsyringe';
+import { getSortedMemberRolesString } from './getSortedMemberRoles';
+
+export async function handleThreadManagement(
```

Comment:
```suggestion
export async function openThread(
```

Unless I'm missing something this would be more fitting naming no?